src/core/components/courses/CourseCell.vue

```diff
@@ -115,14 +110,14 @@ export default {
 
       if (course.value.status === FORTHCOMING) {
         const firstSlot = course.value.slots[0];
-        const rangeToNextDate = moment(firstSlot[0].startDate).diff(moment().startOf('day'), 'd');
+        const rangeToNextDate = getDiffInDays(firstSlot[0].startDate, CompaniDate().startOf('day'));
 
         return rangeToNextDate ? `Commence dans ${formatQuantity('jour', rangeToNextDate)}` : 'Commence aujourd’hui';
       }
 
       if (course.value.status === COMPLETED) {
         const lastSlot = course.value.slots[course.value.slots.length - 1];
-        const rangeToLastDate = moment().endOf('day').diff(moment(lastSlot[0].startDate), 'd');
+        const rangeToLastDate = getDiffInDays(CompaniDate().endOf('day'), lastSlot[0].startDate);
```

Comment:
idem ici mais pour le endof('day'). (Qu'est-ce qui fait que l'on ne compare pas aussi avec le start of mais plutot avec le endof ? 🤔 )

Reply:
je sais pas, j'ai changé